src/jit/gentree.cpp

```diff
@@ -8576,51 +8602,27 @@ void GenTreeUseEdgeIterator::MoveToNextCallUseEdge()
                 break;
 
             case 1:
-            case 3:
+            case 2:
```

Comment:
Yes, that could definitely use a comment or two, though unrelated to this change. It's really just an operand index, so m_index might be a better name.

---
In reply to: [76495055](https://github.com/dotnet/coreclr/pull/6932#discussion_r76495055) [](ancestors = 76495055)